pkg/database/database.go

```diff
@@ -8,9 +8,11 @@ import (
 	glog "log"
 	"net/url"
 	"os"
+	"regexp"
 	"strings"
 	"time"
 
+	goSqlite3 "github.com/mattn/go-sqlite3"
```

Comment:
the default name of `sqlite3` could be used instead

Reply:
done.